.github/workflows/bokehjs-ci.yml

```diff
@@ -97,6 +97,13 @@ jobs:
             cp ${SRC}/report.{json,out} ${CHANGED} ${DST}
           fi
 
+      - name: Upload package
+        if: runner.os == 'Linux' && (success() || failure())
+        uses: actions/upload-artifact@v3
+        with:
+          name: bokehjs-package
+          path: bokehjs/build/dist/bokeh-bokehjs-*.tgz
```

Comment:
Is this right to be different to that used in `bokeh-ci.yml`?:
```
path: bokehjs/build/dist/bokeh-bokehjs.tgz
```

Reply:
This was intentional, but I changed this to have the same artifact layout in bokeh-ci and bokehjs-ci. Now I create a symbolic link to the versioned package, so that tests can resolve it without a version specification.

Reply:
Thanks for clarifying.